src/wifi.rs

```diff
@@ -1442,3 +1445,144 @@ impl WifiWait {
         }
     }
 }
+
+#[cfg(feature = "nightly")]
```

Comment:
And again: you **don't** need `nightly` in the `AsyncWifiDriver`, as it does not use ANY nightly features! That's the whole point why I wanted plain Ole pub async functions on the driver - because they do work in the stable compiler. You can hide it behind `experimental`, but that's orthogonal to `nightly`. The one thing you need nightly for is the `impl Future` trick that you have to use when instantiating your future GATs when implementing the async Wifi trait. So only the async WiFi trait implneeds to be hidden behind nightly, nothing else.